### PR TITLE
DEVOPS-957 Terratests

### PR DESCRIPTION
**Pull request by seanpascual**
# Description

Please explain the changes you made here and link to any relevant issues.

### CODE REVIEW

**Review by marcincuber — CHANGES_REQUESTED** [review]

File: outputs.tf [hunk 1]
```diff
@@ -17,3 +17,28 @@ output "web_acl_capacity" {
...
+
+output "web_acl_assoc_resource_arn" {
+  description = "The ARN of the ALB attached to the Web ACL Association"
+  value       = join("", aws_wafv2_web_acl_association.main.*.resource_arn)
+}
+
+output "web_acl_assoc_acl_arn" {
+  description = "The ARN of the Web ACL attached to the Web ACL Association"
+  value       = join("", aws_wafv2_web_acl_association.main.*.web_acl_arn)
+}
```
[the remark this answers is not in the document]
Comment (seanpascual) [reply outputs.tf:59]: Added

File: .github/workflows/terraform-checks.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-latest
+    name: Terraform Validate
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.15.4
```
Comment (Ohid25) [inline .github/workflows/terraform-checks.yaml:22]: can be set to 0.15.5 now 😄

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: eu-west-1
+        role-skip-session-tagging: true
+        role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+        role-duration-seconds: 1800
+
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.15.4
```
Comment (Ohid25) [inline .github/workflows/test.yaml:50]: here too

Comment (Ohid25) [reply .github/workflows/test.yaml:50]: or even set a global variable here that controls the version

File: .github/workflows/test.yaml [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+      working-directory: ./test/bytematch-test
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16.3
```
Comment (Ohid25) [inline .github/workflows/test.yaml:70]: probably same with go also

**Review by Ohid25 — APPROVED** [review]

**Review by marcincuber — APPROVED** [review]

**Pull request merged**